### DISCUSSION TITLE
Hot reload on microfrontend

### DISCUSSION THREAD

**Discussion by vitorlofonseca** [originjs/vite-plugin-federation#300] [Q&A]
Hello!

I'm working on a [proof of concept](https://github.com/vitorlofonseca/microfrontend-and-third-components-vue) with Vue3 and Vite, to show how microfrontend works with the previous technologies and vite-plugin-federation. What happens is that I'm having an impediment that is: apparently it's not possible to have hot reload in the microfrontend "child" (remote) in development mode. I tried to use `vite build --watch` with `vite preview`, but it doesn't work as well. Do you know how (and even if it's possible) to have hot reload on the microfrontend remote?

**Comment by bayasdev** [comment]
+1
